sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -175,7 +175,10 @@ public void setUser(@Nullable User user) {
    *
    * @param fingerprint the fingerprint list
    */
-  public void setFingerprint(@NotNull List<String> fingerprint) {
+  public void setFingerprint(final @NotNull List<String> fingerprint) {
+    if (fingerprint == null) {
```

Comment:
`m`: What about throwing `IllegalArgumentException`?

Reply:
we cant do that
https://develop.sentry.dev/sdk/philosophy/#degrade-gracefully
this function is called by the users, not the SDK itself